<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KooMainWindow</class>
 <widget class="QMainWindow" name="KooMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Koo: OpenObject KDE Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="common.qrc">
    <normaloff>:/images/koo-icon.png</normaloff>:/images/koo-icon.png</iconset>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="uiServerInformation">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="uiUserName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Requests:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="uiRequests">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushReadRequests">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>Read My Requests</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="common.qrc">
               <normaloff>:/images/read-request.png</normaloff>:/images/read-request.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushSendRequest">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>Send New Request</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="common.qrc">
               <normaloff>:/images/send-request.png</normaloff>:/images/send-request.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuForm">
    <property name="title">
     <string>F&amp;orm</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionMassiveUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionSwitch"/>
    <addaction name="separator"/>
    <addaction name="actionStoreViewSettings"/>
    <addaction name="actionAccessLog"/>
    <addaction name="actionGoToResourceId"/>
    <addaction name="separator"/>
    <addaction name="actionAttach"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <widget class="QMenu" name="menuDatabases">
     <property name="title">
      <string>&amp;Databases</string>
     </property>
     <addaction name="actionNewDatabase"/>
     <addaction name="actionRestoreDatabase"/>
     <addaction name="actionBackupDatabase"/>
     <addaction name="actionDropDatabase"/>
     <addaction name="separator"/>
     <addaction name="actionAdminPassword"/>
    </widget>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="menuDatabases"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>&amp;User</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionClearCache"/>
    <addaction name="separator"/>
    <addaction name="actionSendRequest"/>
    <addaction name="actionReadMyRequest"/>
    <addaction name="separator"/>
    <addaction name="actionWaitingRequests"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>&amp;Reports</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBrowse">
    <property name="title">
     <string>&amp;Browse</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionFullTextSearch"/>
    <addaction name="separator"/>
    <addaction name="actionOpenMenuTab"/>
    <addaction name="actionOpenHomeTab"/>
    <addaction name="separator"/>
    <addaction name="actionNextTab"/>
    <addaction name="actionPreviousTab"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <widget class="QMenu" name="menuDocumentation">
     <property name="title">
      <string>Documentation</string>
     </property>
     <addaction name="actionHtmlManual"/>
     <addaction name="actionPdfManual"/>
     <addaction name="actionDocOpenErpCom"/>
    </widget>
    <addaction name="menuDocumentation"/>
    <addaction name="actionTips"/>
    <addaction name="actionShortcuts"/>
    <addaction name="separator"/>
    <addaction name="actionLicense"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuUser"/>
   <addaction name="menuForm"/>
   <addaction name="menuReports"/>
   <addaction name="menuActions"/>
   <addaction name="menuBrowse"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="uiToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionFullTextSearch"/>
   <addaction name="actionAttach"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="actionSwitch"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string comment="Ctrl+O to Login">Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionNewDatabase">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Database</string>
   </property>
  </action>
  <action name="actionRestoreDatabase">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Restore Database</string>
   </property>
  </action>
  <action name="actionBackupDatabase">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Backup Database</string>
   </property>
  </action>
  <action name="actionDropDatabase">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/trash.png</normaloff>:/images/trash.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Drop Database</string>
   </property>
  </action>
  <action name="actionAdminPassword">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/edit-user.png</normaloff>:/images/edit-user.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Administrator Password</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionSendRequest">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/send-request.png</normaloff>:/images/send-request.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Send Request</string>
   </property>
  </action>
  <action name="actionReadMyRequest">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/read-request.png</normaloff>:/images/read-request.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Read My Requests</string>
   </property>
  </action>
  <action name="actionWaitingRequests">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/read-request.png</normaloff>:/images/read-request.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Waiting Requests</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/duplicate.png</normaloff>:/images/duplicate.png</iconset>
   </property>
   <property name="text">
    <string>D&amp;uplicate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Alt+PgDown</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="shortcut">
    <string>Alt+PgUp</string>
   </property>
  </action>
  <action name="actionSwitch">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/switch_view.png</normaloff>:/images/switch_view.png</iconset>
   </property>
   <property name="text">
    <string>Switch &amp;View</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionOpenMenuTab">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/duplicate.png</normaloff>:/images/duplicate.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Menu Tab</string>
   </property>
  </action>
  <action name="actionNextTab">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
  </action>
  <action name="actionPreviousTab">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAccessLog">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/text_block.png</normaloff>:/images/text_block.png</iconset>
   </property>
   <property name="text">
    <string>View Access Log</string>
   </property>
  </action>
  <action name="actionStoreViewSettings">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Store View Settings</string>
   </property>
  </action>
  <action name="actionGoToResourceId">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/relate.png</normaloff>:/images/relate.png</iconset>
   </property>
   <property name="text">
    <string>Go To Resource ID</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/reload.png</normaloff>:/images/reload.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload/Undo</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/document-export.png</normaloff>:/images/document-export.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xport Data</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/document-import.png</normaloff>:/images/document-import.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Data</string>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/exec.png</normaloff>:/images/exec.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Execute Plugins</string>
   </property>
  </action>
  <action name="actionContextualHelp">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Contextual Help</string>
   </property>
  </action>
  <action name="actionTips">
   <property name="text">
    <string>&amp;Tips</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/key_bindings.png</normaloff>:/images/key_bindings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Keyboard Shortcut</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/koo-icon.png</normaloff>:/images/koo-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionFullTextSearch">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/full_text_search.png</normaloff>:/images/full_text_search.png</iconset>
   </property>
   <property name="text">
    <string>Full &amp;Text Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAttach">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/attach.png</normaloff>:/images/attach.png</iconset>
   </property>
   <property name="text">
    <string>Attachments</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionFormDesigner">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/switch_view.png</normaloff>:/images/switch_view.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Form designer</string>
   </property>
  </action>
  <action name="actionMenuClicked">
   <property name="text">
    <string>&amp;Menu</string>
   </property>
  </action>
  <action name="actionUser_Manual">
   <property name="text">
    <string>User Manual</string>
   </property>
  </action>
  <action name="actionKooManual">
   <property name="text">
    <string>Koo</string>
   </property>
  </action>
  <action name="actionOpenHomeTab">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/gohome.png</normaloff>:/images/gohome.png</iconset>
   </property>
   <property name="text">
    <string>Open Home Tab</string>
   </property>
  </action>
  <action name="actionClearCache">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/trash.png</normaloff>:/images/trash.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear Cache</string>
   </property>
  </action>
  <action name="actionMassiveUpdate">
   <property name="icon">
    <iconset resource="common.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Modify All Selected Records</string>
   </property>
  </action>
  <action name="actionPdfManual">
   <property name="text">
    <string>PDF Manual</string>
   </property>
  </action>
  <action name="actionHtmlManual">
   <property name="text">
    <string>HTML Manual</string>
   </property>
  </action>
  <action name="actionDocOpenErpCom">
   <property name="text">
    <string>doc.openerp.com</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="common.qrc"/>
 </resources>
 <connections/>
</ui>
